integrations/GA4/utils.js

```diff
@@ -96,6 +98,56 @@ function hasRequiredParameters(props, eventMappingObj) {
   return true;
 }
 
+/**
+ *
+ * @param {*} rootObj
+ * @param {*} destination
+ * @param {*} exclusionFields
+ * @returns screens custom variables from rootObj by excluding exclusionFields and adds 
+ * those to destination object
+ */
+function extractCustomVariables(rootObj, destination, exclusionFields) {
+  const mappingKeys = [];
+  Object.keys(rootObj).map((k) => {
+    if (!exclusionFields.includes(k)) mappingKeys.push(k);
+  });
```

Comment:
```suggestion
  const mappingKeys = lodash.intersection(Object.keys(rootObj),exclusionFields);
  
```

Reply:
Basically here we are looking for the negation of intersection. For example 
```
// arrays to be inspected
const array1 = ['a', 'b', 'c'];
const array2 = ['a', 'b'];

// find the common values
const result = _.intersection(array1, array2);

// print result on console
console.log(result);
```

The result is : ["a", "b"]

but we want it to be "c"

Reply:
I think we can use the [difference](https://www.geeksforgeeks.org/lodash-_-difference-function/) then.

Reply:
got it